nss-evening-cohort-14/SweetTooth
# Basic User Auth w/ Firebase (Clientside only)

## Description
- added auth folder, auth buttons into navbar
- added signin/signout functions to auth.js
- added auth code for signin useEffect to App/index.js


## Related Issue
#87





## Motivation and Context

User wants to be able to save their orders, so they need basic authentication to interact with app

## How Can This Be Tested?

git fetch origin `jdc-ui-firebase-auth`
- npm start, then check your React Components browser plugin to confirm:
- User prop state would change from null -> false -> to having your google account info on pressing sign in

## Screenshots (if appropriate):

## Types of changes
